.ruby-version

```diff
@@ -1 +1 @@
-2.7.4
+2.7.7
```

Comment:
we're bumping ruby from 2.7.4 to 2.7.7 on `guest-server`. So this is to make all repos use the same ruby version